api/src/xyz/d1snin/api/modules/RandomModule.kt

```diff
@@ -0,0 +1,45 @@
+package xyz.d1snin.api.modules
+
+import io.ktor.application.*
+import io.ktor.html.*
+import io.ktor.routing.*
+import kotlinx.css.*
+import kotlinx.html.*
+import org.koin.ktor.ext.inject
+import xyz.d1snin.api.controllers.api.RandomController
+
+private val random get() = (0..255).random()
+
+fun Routing.randomModule() {
+
+    val controller by inject<RandomController>()
+
+    get("/random") {
+        val rawParam = call.request.queryParameters["raw"].toBoolean()
+
+        call.respondHtml {
```

Comment:
do you really need to use kotlinx for this? multiline string literals works too imo

Reply:
if you are talking about DSL, then yes, it is very convenient and concise in the first place

Reply:
cope